lib/Dialect/FIRRTL/Transforms/DropName.cpp

```diff
@@ -38,10 +38,12 @@ struct DropNamesPass : public DropNameBase<DropNamesPass> {
 private:
   size_t dropNamesIf(llvm::function_ref<bool(FNamableOp)> pred) {
     size_t changedNames = 0;
+    auto droppableNameAttr =
+        NameKindEnumAttr::get(&this->getContext(), NameKindEnum::DroppableName);
```

Comment:
Why the `this->` portion, could this just be `&getContext()`?

Reply:
Good points, thanks!